skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md

```diff
@@ -408,13 +394,19 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -Description
-Enables administrators to provide explanatory text about the calling policy. For example, the Description might indicate the users the policy should be assigned to.
+### -SpamFilteringEnabledType
+Determines Spam filtering mode.
+
+Possible values:
+- Enabled - Spam Filtering is fully enabled. Both Basic and Captcha IVR checks are performed. In case the call is considered as spam, user will get "Spam Likely" notification in Teams
+- Disabled - Spam Filtering is completely disabled. No checked are performed. "Spam Likely" notification will not appear.
+- EnabledWithoutIVR - Spam Filtering is enabled partially. Captcha IVR (Interactive Voice Response) checks are disabled. "Spam Likely" notification will appear. A call might get dropped in case it gets a high score from Basic checks.
```

Comment:
```suggestion
- Enabled: Spam Filtering is fully enabled. Both Basic and Captcha Interactive Voice Response (IVR) checks are performed. In case the call is considered as spam, the user will get a "Spam Likely" notification in Teams.
- Disabled - Spam Filtering is completely disabled. No checks are performed. A "Spam Likely" notification will not appear.
- EnabledWithoutIVR - Spam Filtering is partially enabled. Captcha IVR checks are disabled. A "Spam Likely" notification will appear. A call might get dropped if it gets a high score from Basic checks.
```